lib/ansible/modules/windows/win_scheduled_task.py

```diff
@@ -99,4 +119,46 @@
     state: present
     enabled: yes
     user: SYSTEM
+
+# Create a scheduled task to run a PowerShell script
+# This task runs as the NETWORK SERVICE user at the highest level of user rights for the account
+- win_scheduled_task:
+    name: TaskName2
+    description: Run a PowerShell script
+    executable: C:\Windows\System32\WindowsPowerShell\v1.0\powershell.exe
+    arguments: -ExecutionPolicy Unrestricted -NonInteractive -File C:\TestDir\Test.ps1
+    time: "6pm"
```

Comment:
Should not be quoted. It might confuse people to think this is needed. It is not.